expyfun/stimuli/tests/test_tracker.py

```diff
@@ -205,19 +227,107 @@ def test_tracker_dealer():
     # bad rand type
     trackers = [TrackerUD(None, 1, 1, 0.06, 0.02, 20, 50, 1)
                 for _ in range(2)]
-    pytest.raises(TypeError, TrackerDealer, trackers, rand=1)
+    with pytest.raises(TypeError, match="missing 1 required positional"):
```

Comment:
Needs to be more general, maybe just `match='argument'`?

https://travis-ci.org/LABSN/expyfun/jobs/599778799#L1538

Reply:
Right - I wasn't sure how much to put for this one and it seemed to pass locally when I ran the pytest. I've updated it to just match "argument"

Reply:
Thanks for catching all theses - sorry there have been so many revisions. First time doing this!

Reply:
No problem, all part of the process -- thanks for sticking with it

Reply:
It's been a fun project!